src/status_im/ui/screens/profile/views.cljs

```diff
@@ -160,66 +163,61 @@
                         :empty-value?        phone-empty?
                         :accessibility-label :profile-phone-number}]))
 
-(defn network-settings []
-  [react/touchable-highlight
-   {:on-press #(dispatch [:navigate-to :network-settings])}
-   [react/view styles/network-settings
-    [react/text {:style styles/network-settings-text}
-     (label :t/network-settings)]
-    [vector-icons/icon :icons/forward {:color :gray}]]])
+(defn settings-title [title]
+  [react/text {:style styles/profile-settings-title}
+   title])
 
-(defn offline-messaging-settings []
+(defn settings-item [label-kw value action-fn active?]
   [react/touchable-highlight
-   {:on-press #(dispatch [:navigate-to :offline-messaging-settings])}
-   [react/view styles/offline-messaging-settings
-    [react/text {:style styles/offline-messaging-settings-text}
-     (label :t/offline-messaging-settings)]
-    [vector-icons/icon :icons/forward {:color :gray}]]])
+   {:on-press action-fn
+    :disabled (not active?)}
+   [react/view styles/settings-item
+    [react/text {:style styles/settings-item-text}
+     (label label-kw)]
```

Comment:
Can we use `i18n/label` ?

Reply:
It's not just `i18n` - there's a ton of non-aliased namespaces, I didn't want to pollute the diff with refactor changes.

Reply:
I am ok with that kind of pollution done step by step. Then gradually we will have everything cleaned!

Reply:
agree with @jeluard , if you're reworking this ns ,lets rework all refers too

Reply:
Cool with me. It think that guideline was added to make code reviews easier